util/genrest/goviewcreator.go

```diff
@@ -290,7 +290,12 @@ func NewView(model *gomodel.Model) (*goview.View, error) {
 	// add '\n' to the "ExtremeValues" ComplianceGroup in compliance_suite.json.
 
 	for _, handler := range registered {
-		file.P(`  router.HandleFunc(%q, rest.%s).Methods(%q)`, handler.pattern, handler.function, handler.verb)
+		// Java's PATCH requests are sent as POST requests with `x-http-methods-override` header
+		if handler.verb == "PATCH" {
+			file.P(`  router.HandleFunc(%q, rest.%s).Methods(%q, %q)`, handler.pattern, handler.function, handler.verb, "POST")
+		} else {
+			file.P(`  router.HandleFunc(%q, rest.%s).Methods(%q)`, handler.pattern, handler.function, handler.verb)
+		}
```

Comment:
```suggestion
                 file.P(`  router.HandleFunc(%q, rest.%s).Methods(%q)`, handler.pattern, handler.function, handler.verb)
		if handler.verb == "PATCH" {
			file.P(`  router.HandleFunc(%q, rest.%s).HeaderRegexp("X-Http-Method-Override", "PATCH").Methods("POST")`, handler.pattern, handler.function)
		}
```

I think we want to have two routes, one for Patch, one for POST w/x-http-method-override.

Can you see if that works? Check the docs for [gorilla/mux](https://pkg.go.dev/github.com/gorilla/mux) if I gave you the wrong incantation.

Reply:
Ah I misunderstood, let me try with two handlers.

Reply:
Thanks for the suggestion. Tested this locally with the ComplianceSuite and it passes with the changes.